src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -269,13 +269,7 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
                 // Then we want to compute the indentation relative to the construct that the collection expression is
                 // attached to.  So ask to be relative to the start of the line the prior token is on if we're on the
                 // same line as it.
-                var option = IndentBlockOption.RelativeToFirstTokenOnBaseTokenLine;
-
-                var firstToken = node.GetFirstToken(includeZeroWidth: true);
-                var lastToken = node.GetLastToken(includeZeroWidth: true);
-                var baseToken = firstToken.GetPreviousToken(includeZeroWidth: true);
-
-                SetAlignmentBlockOperation(list, baseToken, firstToken, lastToken, option);
```

Comment:
An attempt that fixes the failing `CollectionExpressionWrappingTests`

```cs
if (node.Parent is not SwitchExpressionArmSyntax)
{
    SetAlignmentBlockOperation(list, baseToken, firstToken, lastToken, option);
}
```

But I have a feeling this is not the right way :)

Reply:
this seems generally odd to me.  i would expect taht indent block operations would be set for the *switch* expression, and its arms.  Nothing to do with list expressions.

Reply:
Switch has indent block operations but not his arms. I think they don't need to have.

I think the problem is having wrong base & first tokens for list pattern

```c#
var result1 = new int[] { 1, 2 } switch // List pattern
{
    [1] => "one",
    _ => "unknown"
};

var result2 = new int[] { 1, 2 } switch // property pattern clause
{
    { Length: 0 } => "empty",
    [1] => "one",
    _ => "unknown"
};
```

| First Switch expression arm | Base Token | Start Token | End Token
| --- | --- | --- | --- |
| Property pattern clause | `{` (opening prop pattern) | `Length` | `}` (closing prop pattern) |
| List pattern expected | `[` (opening list pattern) | `1` | `]` (closing list pattern) |
| List pattern actual | `{` (opening switch expression) | `[` | `]` (closing list pattern) |